### ISSUE TITLE
Update import `github.com/lyft/proto-gen-validate/validate` to use `envoyproxy`

### ISSUE THREAD

**Issue by nagytech** [pseudomuto/protoc-gen-doc#387]
When running `go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc` I get the following error:

> package github.com/lyft/protoc-gen-validate/validate: code in directory /go/src/github.com/lyft/protoc-gen-validate/validate expects import "github.com/envoyproxy/protoc-gen-validate/validate"

The example can be reproduced using the `Dockerfile` below:

```
FROM golang:1.11-alpine
  
RUN apk add --no-cache git bash
RUN apk -U --no-cache add protobuf

RUN go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc

WORKDIR /app
```

I have a feeling that the import statements like the one shown below are failing due to the redirect from `github.com/lyft` to `github.com/envoyproxy`.

https://github.com/pseudomuto/protoc-gen-doc/blob/ec3fcd0059e127646b1197e118c3c0dab6b418f6/extensions/lyft_validate/lyft_validate_test.go#L5

Any help?

**Comment by nagytech** [comment]
I tested by forking protoc-gen-doc and protokit to the following:

- https://github.com/nagytech/protoc-gen-doc
- https://github.com/nagytech/protokit

After renaming everything adn referencing my go packages, I no longer get the issue.

**Comment by pseudomuto** [comment]
Closed by #388
